src/components/common/Extensions/UserNavigation/partials/UserSubmenu/types.ts

```diff
@@ -8,7 +10,10 @@ export interface UserSubmenuItems {
 
 export interface UserSubmenuItem {
   id: string;
-  label: string;
+  label: MessageDescriptor;
```

Comment:
We discussed earlier to move the formatting outside of the components. Maybe you can do that here as well (sorry, this essentially reverts this).